.github/workflows/wasm-custom_marian-ubuntu.yml

```diff
@@ -2,9 +2,9 @@ name: WASM (Custom Marian) Ubuntu
 
 on:
   push:
-    branches: [ main ]
+    branches: [ main, ci-sandbox ]
```

Comment:
I didn't folow the discussion  beforehand, but why do we have the separate extra branch here (ci-sandbox)?

Reply:
So far I can only trial and error through GitHub CI (no SSH or shell) to get a script working. This (a) pollutes commit history (b) spams people subscribed to PR while I do edits here. After enough of this, I figured it's easier to leave another branch permanently where CI can be triggered by commits, and branch also allows force pushes (not main, for obvious reasons).

In essence use the branch if you can't pull request against main and trigger CI.